tests/python/gaia-ui-tests/gaiatest/visual_components/visual_component.py

```diff
@@ -0,0 +1,7 @@
+from gaiatest.apps.base import PageRegion
+
+
+class VisualComponent(PageRegion):
```

Comment:
this would be great, btw, is there a reason to use js script rather than something like self.is_custom_element_checked(switch) (in gaia-switch) or self.is_element_checked(switch)?